docs.md

```diff
@@ -1410,6 +1410,7 @@ This component's behavior is largely determined by the [TableColumn][103] compon
 *   `controlled` **[Boolean][144]** A flag to disable sorting on all columns, while keeping the sorting arrows. Used when sorting is controlled by an external source. (optional, default `false`)
 *   `onChange` **[Function][143]?** A callback that will be fired when the sorting state changes
 *   `rowComponent` **[Function][143]?** A custom row component for the table. Will be passed the `data` for the row, several internal table states (the current column being sorted (sortPath), whether ascending sort is active or not (ascending), the sorting function (sortFunc), and the value getter (valueGetter)) as well as `children` to render.
+*   `rowComponentProps` **[Object][148]?** Additional props to pass to the rowComponent, useful for passing event handlers
```

Comment:
This is _bit_ pedantic, but I'd prefer to call this `rowProps`. Otherwise, it makes me believe that I **have** to specify a `rowComponent` in order to receive those props, which isn't the case in the current implementation.